src/cascadia/TerminalControl/TermControl.xaml

```diff
@@ -0,0 +1,87 @@
+    mc:Ignorable="d"
+    HorizontalAlignment="Stretch"
+    VerticalAlignment="Stretch"
+    d:DesignHeight="768"
+    d:DesignWidth="1024"
+    TabNavigation="Cycle"
+    IsTabStop="True"
+    AllowFocusOnInteraction="True"
+    AllowDrop="True"
+    Drop="_DragDropHandler"
+    DragOver="_DragOverHandler"
+    Tapped="_TappedHandler"
+    PointerWheelChanged="_MouseWheelHandler"
+    PreviewKeyDown="_KeyDownHandler"
+    CharacterReceived="_CharacterHandler"
+    GotFocus="_GotFocusHandler"
+    LostFocus="_LostFocusHandler">
+        <!--// No matter what order these guys are in, The KeyDown's will fire
+        //      before the CharacterReceived, so we can't easily get characters
+        //      first, then fallback to getting keys from vkeys.
+        // TODO: This apparently handles keys and characters correctly, though
+        //      I'd keep an eye on it, and test more.
+        // I presume that the characters that aren't translated by terminalInput
+        //      just end up getting ignored, and the rest of the input comes
+        //      through CharacterReceived.
+        // I don't believe there's a difference between KeyDown and
+        //      PreviewKeyDown for our purposes
+        // These two handlers _must_ be on this, not _root.-->
+
+    <Grid x:Name="RootGrid">
+        <Image x:Name="BackgroundImage" />
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+
+            <Grid Grid.Column="0">
+                <SwapChainPanel x:Name="SwapChainPanel"
+                                SizeChanged="_SwapChainSizeChanged"
+                                CompositionScaleChanged="_SwapChainScaleChanged"
+                                PointerPressed="_PointerPressedHandler"
+                                PointerMoved="_PointerMovedHandler"
+                                PointerReleased="_PointerReleasedHandler" />
+
+                <!-- Putting this in a grid w/ the SwapChainPanel
+                ensures that it's always aligned w/ the scrollbar -->
+                <local:SearchBoxControl x:Name="SearchBox"
+                                        x:Load="False"
+                                        Visibility="Collapsed"
+                                        HorizontalAlignment="Right"
+                                        VerticalAlignment="Top"
+                                        Search="_Search"
+                                        Closed="_CloseSearchBoxControl" />
+            </Grid>
+
+            <ScrollBar Grid.Column="1"
+                       x:Name="ScrollBar"
+                       Orientation="Vertical"
+                       IndicatorMode="MouseIndicator"
+                       HorizontalAlignment="Right"
+                       VerticalAlignment="Stretch"
+                       Maximum="1"
+                       ViewportSize="10"

```

Comment:
Do we need these? In `TermControl::_InitializeTerminal`, you set these.

Reply:
Good and reasonable defaults are useful too